src/Uno.XamlMerge.Tests/Given_BatchMergeXaml.cs

```diff
@@ -148,6 +148,149 @@ public void When_Different_Namespace_Should_Be_Merged_With_Members()
         ValidateOutput(task);
     }
 
+    [TestMethod]
+    public void When_Different_Namespace_Should_Be_Merged_Multiple_Attributes()
+    {
+        var task = CreateMerger();
+
+        Assert.IsTrue(task.Execute());
+
+        ValidateOutput(task);
+    }
+
+#if false // Can be good for local testing against Uno repository.
```

Comment:
Not sure if this can be useful to anyone? It helped me track down the issue that is not caught by tests but reproduces in Uno.UI.

I can delete this test if it's not much useful.

Reply:
If the issue is about escaping, can you add a test that validates that specific part?

Reply:
Escaping was one of the issues. Added a test.